lib/generators/active_versioning/templates/migrations/create_versions.rb

```diff
@@ -1,4 +1,9 @@
-class CreateVersions < ActiveRecord::Migration
+<%
+  parent_class = ActiveRecord::Migration
+  parent_class = parent_class[parent_class.current_version] if Rails::VERSION::MAJOR >= 5
```

Comment:
does this end up formatted like `ActiveRecord::Migration[5.0]`? just curious, I took a quick glance at some docs and wanted to confirm

Reply:
I get `ActiveRecord::Migration::Current` when I run this in my rails console.

Reply:
Here's an example of how this was handled in another gem: https://github.com/paper-trail-gem/paper_trail/pull/950/files

Reply:
@dbernheisel yeah, thats the expected format.

@cwmanning I've seen migrations from other generators that end up like that, but I'm not sure which is preferred.

Reply:
I don't think you want `ActiveRecord::Migration::Current`, because as a person upgrades their rails versions, so do their migrations which is not what we intend. It needs to be a migration that never evolves after it's generated.

You want something like `ActiveRecord::Migration[5.0]`